docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -169,15 +169,16 @@ The operation was successful if the returned status says `true`.
 
 1. Using the left navigation, go back to **Workloads** > **Functions**.
 2. Select your `lastorder` Function and navigate to the **Configuration** tab.
```

Comment:
```suggestion
1. Go back to your `lastorder` Function's view in Kyma Dashboard.
```
You never really navigate away from it Dashboard-wise, so it's a bit confusing to have these steps repeated here.